Podverse/PodcastSearchResult.swift

```diff
@@ -8,15 +8,75 @@
 
 import Foundation
 
-class SearchResultPodcast {
-    var artistName: String?
-    var episodesTotal: Int?
-    var feedUrl: String?
-    var imageURL: String?
-    var itunesImageURL: String?
-    var isSubscribed: Bool?
-    var lastPubDate: Date?
+class PodcastSearchResult {
+    var id: String?
+    var buzzScore: String?
     var categories: String?
-    var summary: String?
+    var description: String?
+    var hosts: String?
+    var imageUrl: String?
+    var network: String?
+    var rssUrl: String?
+    var scFeed: String?
     var title: String?
+    var webProfiles: String?
+    
+    
+    static func convertJSONToSearchResult (json: AnyObject) -> PodcastSearchResult? {
+        let podcast = PodcastSearchResult()
+        
+        podcast.id = json["id"] as? String
+        
+        podcast.buzzScore = json["buzz_score"] as? String
+        
+        if let categories = json["categories"] as? [[String:Any]] {
+            var categoriesString = ""
+            for category in categories {
+                if let c = category["name"] as? String {
+                    categoriesString += c + ", "
+                }
+            }
+            categoriesString = String(categoriesString.characters.dropLast(2))
+            podcast.categories = categoriesString
+        }
+        
+        podcast.description = json["description"] as? String
+        
+        if let hosts = json["hosts"] as? [String] {
+            var hostsString = ""
+            for host in hosts {
+                hostsString += host + ", "
+            }
+            hostsString = String(hostsString.characters.dropLast(2))
+            podcast.hosts = hostsString
+        }
+        
+        if let imageFiles = json["image_files"] as? [[String:Any]] {
```

Comment:
Would I keep [[String:Any]] in the first part of the line, or somehow use [String:Any] twice in a row or something?